src/ducks/transactions/TransactionsWithSelection.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react'
+import Transactions from './Transactions'
+import TransactionModal from './TransactionModal'
+
+class TransactionsWithSelection extends Component {
```

Comment:
I regrouped `TransactionWithSelection` in a previous commit, it is available here: https://github.com/cozy/cozy-banks/blob/c7470be57fb55fe632e8f75d158ececac683e34e/src/ducks/transactions/Transactions.jsx#L335

I think it is better to have it near Transactions since it does almost nothing. WDYT ?

Reply:
Actually I messed up while rebasing. Your change is here, but I didn't delete `TransactionsWithSelection.jsx`. Done now :+1: :heavy_check_mark: